apply-personal-number

richardwd opened the pull request:
apply-personal-number

richardwd committed:
Add files via upload

person-number-related pictures

richardwd committed:
Delete images/pn directory

richardwd committed:
Add files via upload

richardwd committed:
Add files via upload

Teach how to apply personal number...

richardwd commented:
apply-personal-number

chuanjin commented on apply-personal-number.md:
```diff
@@ -0,0 +1,31 @@
+
+## Step 3:
+##### The officer will check your files and you can fill the website on the computer they will find for you. Fill it step by step.
+![Aaron Swartz](https://raw.githubusercontent.com/richardwd/guidelines/main/images/pn/1.png)
```
Should use relative path here. Refer [here](https://github.com/Orientalent/guidelines/commit/cfd435fb6741b427ad0f17d37ebe4e738268ea61#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R3)

chuanjin commented on apply-personal-number.md:
```diff
@@ -0,0 +1,31 @@
+![Aaron Swartz](https://raw.githubusercontent.com/richardwd/guidelines/main/images/pn/8.png)
+![Aaron Swartz](https://raw.githubusercontent.com/richardwd/guidelines/main/images/pn/9.png)
+
+##### After 1 to 18 months, check the mailbox, and a letter will be sent to you with a personal number and a website. Send the personal number to Ola and cc to CJ by E-mail. 
```
Usually the waiting time is 1 to 3 months, check your mailbox regularly, Tax Agency will send you a letter by post with your personal number. When you recieve the letter, send your personal number to admin@orientalent.se to update your infomation in the company.

richardwd committed:
Document update

1.use relative path
2.Usually the waiting time is 1 to 3 months, check your mailbox regularly, Tax Agency will send you a letter by post with your personal number. When you recieve the letter, send your personal number to admin@orientalent.se to update your infomation in the company.

chuanjin commented on apply-personal-number.md:
```diff
@@ -0,0 +1,31 @@
+
+##### Before taking the photo, you need to pay the card production fee according to the payment method specified in the letter. Approximate 400 kroner.
+
+##### Then go to "Skattekontoret" again to take a picture. After about 2 months, you can get an ID card. Then apply for a bank card with your ID card. Send the bank card number to Ola and cc to CJ by E-mail.
```
Send your bank account number to admin@orientalent.se by E-mail.

richardwd reviewed (commented):
1.use relative path here
2.Usually the waiting time is 1 to 3 months, check your mailbox regularly, Tax Agency will send you a letter by post with your personal number. When you recieve the letter, send your personal number to admin@orientalent.se to update your infomation in the company.

chuanjin commented on apply-personal-number.md:
```diff
@@ -0,0 +1,31 @@
+
+## Step 3:
+##### The officer will check your files and you can fill the website on the computer they will find for you. Fill it step by step.
+![Aaron Swartz](images/pn/1.png)
```
Remove "Aaron Swartz" and all below

richardwd committed:
Update apply-personal-number.md

Send your bank account number to admin@orientalent.se by E-mail.

richardwd reviewed (commented):
Send your bank account number to admin@orientalent.se by E-mail.

richardwd committed:
Update apply-personal-number.md

Remove "Aaron Swartz" and all below

richardwd reviewed (commented):
Remove "Aaron Swartz" and all below
